facilities/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from common.models import AbstractBase, SubLocation
+
+FACILLITY_STATUS = (
```

Comment:
This should be a simple model, with default data.

For this project, apply a simple rule: NO CHOICE FIELDS, UNLESS PREVIOUSLY DISCUSSED AND AGREED.